### DISCUSSION TITLE
Bungeecord Join and Leave messages

### DISCUSSION THREAD

**Discussion by Dziaod** [Zrips/CMI#3359] [Feature Requests]
Bungeecord Join and Leave messages

---
When a player joins the proxy, the join message is displayed on all servers, and when he disconnects from the proxy, the leave message gets displayed on all servers.
On server switch, the join and leave messages are not displayed.

Many servers have multiple servers for the same project, and join/leave messages make players confused why the player in question left, but still talks in chat.

**Comment by mercurialmusic** [comment]
Yeah, this would be nice. I currently use Global Bell (which honestly is great and I probably wouldn't move away from it unless CMI matched it) and have CMI disable the vanilla join/leave messages.
